Sources/InfomaniakCore/Account/Account.swift

```diff
@@ -17,6 +17,7 @@
  */
 
 import Foundation
+import InfomaniakLogin
```

Comment:
If we can use _protocols_ and the _DI_ to remove the new direct dependency from `core` to `InfomaniakLogin` it would be perfect.

And I think we can do it. You need a protocol on `ApiToken` and `InfomaniakNetworkLogin`, or at least the bare minimum that `Account` needs (it looks like just something that can provide a token string)

Reply:
The token is more than a string because it is linked to userId. 
At some point we would need to share the protocol between Login and Core. Where would this be done ? At app level ?

Reply:
The protocol would be in core, as it is a requirement of the model object that lives in core.
There would still be a dependency from login to core, but the login code would actually be in the login package like you refactored.

Ideally the model would be in a `model package` with a dependency on realm, and the login package would have a dependency on the `model package` only.

Reply:
OK discussed.

> At some point we would need to share the protocol between Login and Core. Where would this be done ? At app level ?

Yes, would be more complicated but we can completely cut links between `core` and `login`, at the cost of more complexity (2 protocols, linked in the app.) 

So agreed we can go on like this, and ideally we should split core some more if we have the time down the line.